sql-db/panache-flyway/src/test/java/io/quarkus/ts/sqldb/panacheflyway/dbpool/MySqlDatabaseTestResource.java

```diff
@@ -20,25 +20,36 @@ public class MySqlDatabaseTestResource implements QuarkusTestResourceLifecycleMa
     private static final String DEFAULT_SCHEMA = "test";
     private static final String MYSQL = "mysql";
     private static final String WITH_XA_PROFILE = ".with-xa";
+    private static final String USER_PROPERTY = "MYSQL_USER";
+    private static final String PASSWORD_PROPERTY = "MYSQL_PASSWORD";
+    private static final String PASSWORD_ROOT_PROPERTY = "MYSQL_ROOT_PASSWORD";
+    private static final String DATABASE_PROPERTY = "MYSQL_DATABASE";
+    private static final String USER = "user";
 
-    private MySQLContainer<?> container;
+    private GenericContainer<?> container;
 
     @Override
     public Map<String, String> start() {
         String image = System.getProperty("mysql.80.image");
 
-        container = new MySQLContainer<>(
+        container = new GenericContainer<>(
```

Comment:
I suggest to leave the explanation (about padding) somewhere in the class, so it will be less surprising for future developers